domino_web_example/web/main.dart

```diff
@@ -1,33 +1,48 @@
+import 'dart:async';
 import 'dart:html';
+import 'package:async_tracker/async_tracker.dart';
+
 import 'templates/human.dart';
 import 'templates/mainElem.g.dart';
 import 'package:domino/src/experimental/idom.dart' as idom;
 import 'package:domino/src/experimental/idom_browser.dart' as ib;
 
 void main() {
-  querySelector('#output').text = 'Not changed';
-  final ctx = ib.BrowserDomContext(querySelector('#output'));
-  ctx.text('Text context');
-  ctx.open('div');
-  for (var i = 1; i <= 10; ++i) {
-    ctx.text('value $i\n');
-  }
-  ctx.close();
-  final man = Human('Bob', 300, 'Earth');
-  var clickFun;
-  clickFun = (idom.DomEvent e) {
-    window.alert('Hello from here with $man');
-    man.name = 'Password';
-    man.age = 600;
-    man.location = 'Moon';
-    final ctx = ib.BrowserDomContext(querySelector('#output'));
-    renderMain(ctx, human: man, clickName: 'Click', clickFun: clickFun);
+  final tracker = AsyncTracker();
+  final man = Human('Bob', 10, 'Earth');
+  final output = querySelector('#output');
+  output.text = 'Not changed';
+  void renderAll() {
+    final ctx = ib.BrowserDomContext(output);
+    ctx.text('Text context');
+    ctx.open('div');
+    for (var i = 1; i <= man.age; ++i) {
+      ctx.text('value $i\n');
+    }
+    ctx.close();
+    var clickFun;
+    clickFun = (idom.DomEvent e) {
+      //window.alert('Hello from here with $man');
+      man.name = 'password';
+      man.age = 20;
+      man.location = 'Moon';
+    };
+    renderMain(ctx, human: man, clickName: 'Click!', clickFun: clickFun);
     ctx.close();
+
+    querySelector('#clickButton').on['onClick'].listen((e) {
+      window.alert('Hello from there');
+    });
   };
-  renderMain(ctx, human: man, clickName: 'Click!', clickFun: clickFun);
 
-  querySelector('#clickButton').on['onClick'].listen((e) {
-    window.alert('Hello from there');
+  var canUpdate = false;
+  Timer.periodic(Duration(milliseconds: 50), (t) {canUpdate = true;});
```

Comment:
I assume this is because an event cascade re-triggers the render. It would be worth to investigate which event it is and fix it. Can be in separate PR.